R/get_national_data.R

```diff
@@ -63,6 +64,9 @@ get_national_data <- function(country, source = "who", steps = FALSE,
   # non-specific cleaning and checks
   nation_class$process()
 
+  if (return_class) {
+    return(nation_class)
+  }
```

Comment:
add else and use as a non-exported function as using twice (at least)

Reply:
Is this just removing the `@export` from get_national_data & get_regional_data?

Reply:
no sorry. I mean adding a new internal function to contain these repeated lines of code and then using it in both of these functions.

Reply:
push a commit adding this as I wasn't really explaining myself well. We can circle back at a later date?